config/configmap/inferenceservice.yaml

```diff
@@ -425,7 +425,20 @@ data:
          # ModelMesh https://kserve.github.io/website/master/admin/modelmesh/
          "defaultDeploymentMode": "Serverless"
        }
-     
+
+      annotationsPropagationDisallowList: |-
+        [
+          "autoscaling.knative.dev/min-scale",
+          "autoscaling.knative.dev/max-scale",
+          "internal.serving.kserve.io/storage-initializer-sourceuri",
```

Comment:
I've seen several KServe internal annotations and labels. I so so remember some of those internal annotations/labels would control injections on the pods.

So, I wonder if it is worth to disallow the KServe internal ones even if they are not on the list...